java-frontend/src/test/java/org/sonar/java/bytecode/se/BytecodeEGWalkerExecuteTest.java

```diff
@@ -79,6 +83,25 @@
   private static final Set<Integer> LONG_OPCODE = ImmutableSet.of(LADD, LSUB, LMUL, LDIV, LAND, LOR, LXOR, LREM, LSHL, LSHR, LUSHR,
     DADD, DSUB, DMUL, DDIV, DREM);
 
+  private static final String TRY_CATCH_SIGNATURE = BytecodeEGWalkerExecuteTest.class.getCanonicalName() + "#tryCatch(Z)V";
+  private static final String TRY_WRONG_CATCH_SIGNATURE = BytecodeEGWalkerExecuteTest.class.getCanonicalName() + "#tryWrongCatch(Z)V";
+
+
+  SemanticModel semanticModel;
+  BytecodeEGWalker walker;
+  private SquidClassLoader squidClassLoader;
+
+  public void setUp() {
```

Comment:
I would challenge the name. please refine it:  setup in context of Junit test means something and it is confusing to have a method named liked this and not annotated with @Before